diff --git a/daemon-files/minecraftd.conf b/daemon-files/minecraftd.conf
index 13f3371..7dccd51 100644
--- a/daemon-files/minecraftd.conf
+++ b/daemon-files/minecraftd.conf
@@ -8,13 +8,13 @@ BACKUP_PATHS="world" # World paths separated by spaces relative to SERVER_ROOT
 BACKUP_FLAGS="-z"
 KEEP_BACKUPS="10"
 GAME_USER="minecraft"
-MAIN_EXECUTABLE="minecraft_server.jar"
+MAIN_EXECUTABLE="forge"
 SESSION_NAME="minecraft"

 # System parameters for java
 #  -Xms sets the intial heap size (must be a multiple of 1024 and greater than 2MB, no spaces!)
 #  -Xmx sets the maximum heap size (must be a multiple of 1024 and greater than 2MB, no spaces!)
-SERVER_START_CMD="java -Xms512M -Xmx1024M -jar './${MAIN_EXECUTABLE}' nogui"
+SERVER_START_CMD="@out@/lib/start-server.sh"

 # System parameters for the actual game server
 # Describes whether a daemon process which stops the server if it is not used by a player
diff --git a/daemon-files/minecraftd.sh b/daemon-files/minecraftd.sh
index cd69672..cc14eb1 100755
--- a/daemon-files/minecraftd.sh
+++ b/daemon-files/minecraftd.sh
@@ -32,7 +32,7 @@ declare -r game="minecraft"
 [[ -n "${GAME_COMMAND_DUMP}" ]] && declare -r GAME_COMMAND_DUMP=${GAME_COMMAND_DUMP} || GAME_COMMAND_DUMP="/tmp/${myname}_${SESSION_NAME}_command_dump.txt"

 # Variables passed over the command line will always override the one from a config file
-source /etc/conf.d/"${game}" 2>/dev/null || >&2 echo "Could not source /etc/conf.d/${game}"
+source @out@/etc/"${game}" 2>/dev/null || >&2 echo "Could not source @out@/etc/${game}"

 # Preserve the content of IDLE_SERVER without making it readonly
 [[ -n ${tmp_IDLE_SERVER} ]] && IDLE_SERVER=${tmp_IDLE_SERVER}
